plotter/image.go

```diff
@@ -71,6 +71,45 @@ func (img *Image) GlyphBoxes(plt *plot.Plot) []plot.GlyphBox {
 	return nil
 }
 
+// transform warps the image to align with non-linear axes.
+func (img *Image) transform(p *plot.Plot) image.Image {
```

Comment:
Can you rename this to `transformFor`? The current name suggests the transformation is being done to `p`.